src/ColumnCues.h

```diff
@@ -0,0 +1,51 @@
+#ifndef COLUMN_CUES_H
+#define COLUMN_CUES_H
+
+#include "GameConstantsAndTypes.h"
+class NoteData;
+
+/* ColumnCues are used to indicate to the player which column the next note will occur
+ after a long gap in the stepchart.
+ This info is made available to the theme via lua functions in Steps.cpp
+ */
+struct ColumnCueColumn
+{
+	int colNum;
+	bool isMine;
```

Comment:
Let's do a couple of things:

Instead of having an `isMine` bool, let's just return the `TapNoteType` itself. This will help limit some of the conditionals in the code.

Technically we care about Taps, Hold Heads, Mines, and Lifts (which weren't accounted for before).